### PR TITLE
Mostly done renaming

### PR DESCRIPTION
**Pull request by jzeeck**
See #11

### CODE REVIEW

File: idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java [hunk 1]
```diff
@@ -28,27 +28,27 @@
 import javax.inject.Inject;
 
 /**
- * Command line tool to assign new IDs.
+ * Command line tool to createLabel new IDs.
```
Comment (luuse) [inline idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java]: Update this to be more correct english?

File: idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java [hunk 2]
```diff
@@ -28,27 +28,27 @@
...
   @Inject
-  Assign(final LabelClient labelClient) {
+  CreateLabel(final LabelClient labelClient) {
     this.labelClient = checkNotNull(labelClient);
   }
 
   /**
-   * Entry-point for the assign application. The assign program is normally not executed directly
-   * but rather through the main project.
+   * Entry-point for the createLabel label application. The createLabel program is normally not
```
Comment (luuse) [inline idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java]: Update this to be more correct english?

File: idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java [hunk 3]
```diff
@@ -28,27 +28,27 @@
...
+   * executed directly but rather through the main project.
    *
    * @param args The command-line arguments
    */
   public static void main(final String[] args) {
     final CommandLineApplication application = CommandLineApplication.builder()
-        .command("id assign")
+        .command("id createLabel")
```
Comment (luuse) [inline idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java]: Should probably more something more along the lines of `label create`?

File: idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java [hunk 4]
```diff
@@ -104,11 +104,11 @@ private static LabelType type(final List<?> nonOptionArguments) {
       String stringType = nonOptionArguments.get(0).toString();
       return LabelType.fromValue(stringType);
     } catch (IndexOutOfBoundsException e) {
-      throw new IllegalArgumentException("Missing identifier type to assign");
+      throw new IllegalArgumentException("Missing identifier type to createLabel");
```
Comment (luuse) [inline idmanager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java]: Can you update this to be a more flowing sentence?

File: main/build.gradle [hunk 5]
```diff
@@ -9,7 +9,7 @@ dependencies {
     compile project(":freki-core")
     compile project(":freki-cassandra")
 
-    compile project(":idmanager:freki-assign")
+    compile project(":idmanager:freki-create")
```
Comment (luuse) [inline main/build.gradle]: Can you rename the `idmanager` project to `label-manager` in this PR as well?

File: main/build.gradle [hunk 6]
```diff
@@ -9,7 +9,7 @@ dependencies {
     compile project(":freki-core")
     compile project(":freki-cassandra")
 
-    compile project(":idmanager:freki-assign")
+    compile project(":label-manager:freki-create")
```
Comment (luuse) [inline main/build.gradle]: The `idmanager` folder should be renamed if this is renamed as well

File: label-manager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java [hunk 7]
```diff
@@ -28,27 +28,27 @@
...
-   * but rather through the main project.
+   * Entry-point for the createLabel application. The createLabel program is normally not
+   * executed directly but rather through the main project.
    *
    * @param args The command-line arguments
    */
   public static void main(final String[] args) {
     final CommandLineApplication application = CommandLineApplication.builder()
-        .command("id assign")
+        .command("id label create")
```
Comment (luuse) [inline label-manager/createlabel/src/main/java/se/tre/freki/idmanager/CreateLabel.java]: The string used here should match the path from `Main`

File: main/build.gradle (hunk 6 above)
Comment (luuse) [inline main/build.gradle]: Might be the package that's the problem. There's a bunch of `idmanager` references left though.

File: suppressions.xml [hunk 8]
```diff
@@ -6,7 +6,7 @@
 
 <suppressions>
   <!-- Ignore generated sources -->
-  <suppress checks="." files="(main|grpc|cassandra|core|idmanager[\\/]assign|web)[\\/]build" />
+  <suppress checks="." files="(main|grpc|cassandra|core|idmanager[\\/]createLabel|web)[\\/]build" />
```
Comment (luuse) [inline suppressions.xml]: `label-manager`?

File: suppressions.xml [hunk 9]
```diff
@@ -14,7 +14,7 @@
   <suppress checks="EmptyLineSeparator" files="package-info.java$" />
 
   <!-- Don't require javadocs for test methods -->
-  <suppress checks="JavadocMethod" files="(all|cassandra|core|idmanager[\\/]assign|web)[\\/]src[\\/]test" />
+  <suppress checks="JavadocMethod" files="(all|cassandra|core|idmanager[\\/]createLabel|web)[\\/]src[\\/]test" />
```
Comment (luuse) [inline suppressions.xml]: `label-manager`?

File: web/src/main/java/se/tre/freki/web/HttpServer.java [hunk 10]
```diff
@@ -24,7 +24,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(HttpServer.class);
 
   /**
-   * Entry-point for the http server application. The assign program is normally not executed
+   * Entry-point for the http server application. The create program is normally not executed
```
Comment (luuse) [inline web/src/main/java/se/tre/freki/web/HttpServer.java]: This is a typo. It should not have said assign to begin with. Can you change this to "web"?

File: main/build.gradle (hunk 6 above)
Comment (jzeeck) [inline main/build.gradle]: Yeah package problem

File: label-manager/createlabel/src/main/java/se/tre/freki/labelmanager/CreateLabel.java [hunk 11]
```diff
@@ -28,27 +28,27 @@
...
-   * but rather through the main project.
+   * Entry-point for the createLabel application. The createLabel program is normally not
+   * executed directly but rather through the main project.
    *
    * @param args The command-line arguments
    */
   public static void main(final String[] args) {
     final CommandLineApplication application = CommandLineApplication.builder()
-        .command("id assign")
+        .command("id label create")
```
Comment (luuse) [inline label-manager/createlabel/src/main/java/se/tre/freki/labelmanager/CreateLabel.java]: This should equal the subcommand used to normally invoke this program, e.g. `label create`.

Comment (luuse) [inline label-manager/createlabel/src/main/java/se/tre/freki/labelmanager/CreateLabel.java]: This means that that [subcommand](https://github.com/hi3g/freki/pull/71/files#diff-b0080e7320f8dd7c8d2e3027dd0c67ebR26) will have to renamed as well.

**Comment by luuse** [comment]
Could you squash everything into 1 commit after fixing the final comments ?

File: main/src/main/java/se/tre/freki/Main.java [hunk 12]
```diff
@@ -38,8 +38,8 @@ private static void usage(final String[] args) {
   }
 
   private static void handleIdManager(final String[] args) {
-    if ("assign".equals(args[1])) {
-      Assign.main(Arrays.copyOfRange(args, 2, args.length));
+    if ("label".equals(args[1])) {
```
Comment (luuse) [inline main/src/main/java/se/tre/freki/Main.java]: Should be `create`, not `label`.

**Pull request merged**